src/lib.rs

```diff
@@ -328,7 +326,7 @@ To see all the flags the proxied tool accepts run `cargo-{} -- -help`.{}",
             cargo.args(&["--lib", lib_name]);
             lib_name
         } else {
-            unreachable!();
+            ""
```

Comment:
Why not `None`?

Reply:
Because the other paths resolve to a `&str`, too and I didn't want to change the type to an `Option` and make the additional changes to all the other places using it.